ethToElrond/v2/ethToElrond/steps/temp.go

```diff
@@ -0,0 +1,9 @@
+package steps
+
+// TODO remove this file
+var _ = getPendingStep{}
```

Comment:
what is this for? these are not interface assertions

Reply:
just a method to pass the linter tests that will be (and its already) deleted after the unit tests are added